src/System.Management.Automation/resources/RemotingErrorIdStrings.resx

```diff
@@ -1621,4 +1621,19 @@ All WinRM sessions connected to Windows PowerShell session configurations, such
   <data name="SSHClientEndNoErrorMessage" xml:space="preserve">
     <value>The SSH client session has ended with no error message.</value>
   </data>
+  <data name="MissingRequiredSSHParameter" xml:space="preserve">
+    <value>The provided SSHConnection hash table is missing the required ComputerName or HostName parameter.</value>
```

Comment:
Hashtable is a single word since it is referring to System.Collections.Hashtable